release_notes.md

```diff
@@ -0,0 +1,38 @@
+ # Release notes
+
+- Woof Mobile APP
+
```

Comment:
@iBYarchitect @julyakmita 
What do you think about keeping only the title of the file and the app name, we haven't made any releases yet, so for me, it could be empty.
Or describe the expected MVP and the date of the first release.

Reply:
We haven't released anything so this section should be empty at all

Reply:
Agree. Now the file is keeping only the title of the file and the app name.